app/lib/sanitize_config.rb

```diff
@@ -67,7 +67,7 @@ module Config
     end
 
     UNSUPPORTED_HREF_TRANSFORMER = lambda do |env|
-      return unless env[:node_name] == 'a'
+      return unless env[:node_name] == 'a' and env[:node]['href']
```

Comment:
This one should be kept: nil `href` is an expected case that replaces the link by its contents.